feature/office/src/main/java/com/iteneum/office/presentation/ui/OfficeView.kt

```diff
@@ -28,20 +30,45 @@ import com.iteneum.office.data.OfficeModel
  */
 
 @Composable
-fun OfficeView() {
-    val viewModel = OfficeViewModel()
+fun OfficeView(viewModel: OfficeViewModel = hiltViewModel()) {
 
     LaunchedEffect(true) {
         viewModel.getOfficeInformation()
     }
 
-    val officeData = viewModel.officeInfo
+    val uiState = viewModel.state.collectAsState()
 
-    officeData?.let {
-        OfficeUI(it,
-            onCallButtonClicked = { viewModel.makeCall() },
-            onEmailButtonClicked = { viewModel.sendEmail() })
+    when (uiState.value) {
+        is OfficeStateResponse.Error -> {
+            ShowStateContent(stringResource(R.string.Error))
+        }
+
+        is OfficeStateResponse.Loading -> {
+            ShowStateContent(stringResource(R.string.Loading))
+        }
+
+        is OfficeStateResponse.Success -> {
+            val OfficeData = viewModel.officeInfo
```

Comment:
rename **OfficeData**, first letter should be in lower case, variables should be named in lower case

Reply:
lowered 
![imagen](https://github.com/irv205/droid_leaspert/assets/128720218/76407dfa-eff7-4f1f-9c93-4edf893b6208)